docs/swagger.yml

```diff
@@ -916,6 +971,7 @@ paths:
       summary: "Obtain a user token."
       description: "Obtain a user token."
       operationId: "obtainToken"
+      security: []
```

Comment:
Status endpoint also doesn't require a token

Reply:
Fixed

Reply:
looks like you updated log out instead of status.

Reply:
Fixed.